SongLab/Components/RecordingsList/RecordingCell.swift

```diff
@@ -7,70 +7,211 @@
 
 import SwiftUI
 
+enum theme: Double {
+    case glassy = 0.8
+    case superGlassy = 0.51
+    case opaque = 0.5
+}
+
 struct RecordingCell: View {
-        
+    
     // MARK: - API
     
+    @Binding var appTheme: String
+    
     init(
         currentlyPlaying: Session?,
         session: Session,
+        appTheme: Binding<String>,
         playButtonAction: @escaping (_: Session) -> Void,
         stopButtonAction: @escaping () -> Void,
         trashButtonAction: @escaping (_: Session) -> Void
     ) {
         self.currentlyPlaying = currentlyPlaying
         self.session = session
+        _appTheme = appTheme
         self.playButtonAction = playButtonAction
         self.stopButtonAction = stopButtonAction
         self.trashButtonAction = trashButtonAction
     }
     
     // MARK: - Variables
-        
+    
+    @Environment(\.colorScheme) private var colorScheme
+    
     private var session: Session
     private var currentlyPlaying: Session?
-        
+    
     private let playButtonAction: (_ session: Session) -> Void
     private let stopButtonAction: () -> Void
     private let trashButtonAction: (_ session: Session) -> Void
     
+    private var cellOpacity: Double {
+        colorScheme == .dark ? 0.6 : 1.0
+    }
+    
+    private var cellColor: Color {
+        switch appTheme {
+        case "glass":
```

Comment:
The different themes should definitely be defined using an `enum` instead of strings.